web/src/lib/components/shared-components/full-screen-modal.svelte

```diff
@@ -48,7 +48,7 @@
     role="presentation"
     in:fade={{ duration: 100 }}
     out:fade={{ duration: 100 }}
-    class="fixed left-0 top-0 z-[9990] flex h-screen w-screen place-content-center place-items-center bg-black/40"
+    class="fixed left-0 top-0 z-[9999] flex h-screen w-screen place-content-center place-items-center bg-black/40"
```

Comment:
If 9990 isn't high enough maybe we're getting a little crazy with our z-index numbers :sweat_smile: Or was this just a typo originally?

Reply:
Correct, there is an overlay model for face merge, and the confirmation isn't shown unless it reaches 9999. What do you guys think is is the proper way to handle this z-indexing war?